katas/es6/language/generators/yield.js

```diff
@@ -0,0 +1,38 @@
+// Generators - Yield Statements
+// To do: make all tests pass, leave the assert lines unchanged!
+
+describe('generator instance', () => {
+  
+  function* generator() {
+    yield 'hello';
+    yield 'world';
+  }
+  
+  // create instance
+  let genInstance = generator();
+  
+  it('should have an undefined value property after initialisation', () => {
+    let value = genInstance;
```

Comment:
do you expect `genInstance.value` to be there? that would be trying to access the `value` property of the iterator, does that make sense?